src/js/pages/attendee/Event.jsx

```diff
@@ -27,6 +27,20 @@ export default class Event extends React.Component {
     EventStore.removeListener('error', this.showError);
   }
 
+  componentDidUpdate(prevProps, prevState) {
+    let oldId = -1;
+    let newId = -1;
+    if (prevProps.params.hasOwnProperty('eventId')) {
+      oldId = prevProps.params.eventId;
+    }
+    if (this.props.params.hasOwnProperty('eventId')) {
+      newId = this.props.params.eventId;
+    }
+    if (oldId != newId) {
```

Comment:
i think it was when you nav from one event to another without anything inbetween. either way good practice.